lang/lang.pt-BR.json

```diff
@@ -881,41 +881,41 @@
+            "inviteEmbed": "[Convide {{REF:bot.name}} para o seu servidor!]({{REF:links.invite}})",
             "docs": "https://docs.friendtime.ml",
-            "docsEmbed": "[Documentação]({{REF:links.docs}}/)",
+            "docsEmbed": "[Ver Documentação]({{REF:links.docs}}/)",
             "vote": "https://top.gg/bot/471091072546766849/vote",
-            "voteEmbed": "[Vote em mim {{REF:bot.name}}!]({{REF:links.vote}})",
+            "voteEmbed": "[Vote no {{REF:bot.name}}!]({{REF:links.vote}})",
             "support": "https://discord.gg/GQcBR8e",
-            "supportEmbed": "[Junte ao servidor de suporte]({{REF:links.support}})",
+            "supportEmbed": "[Junte-se ao servidor de suporte]({{REF:links.support}})",
             "stream": "https://www.twitch.tv/novakevin",
             "author": "https://github.com/KevinNovak",
             "authorEmbed": "[Kevin Novak]({{REF:links.author}})",
             "source": "https://github.com/KevinNovak/Friend-Time",
-            "sourceEmbed": "[Verificar o código]({{REF:links.source}})",
+            "sourceEmbed": "[Verificar o código-fonte]({{REF:links.source}})",
             "template": "https://github.com/KevinNovak/Discord-Bot-TypeScript-Template",
-            "templateEmbed": "[Discord Bot TypeScript Template]({{REF:links.template}})",
+            "templateEmbed": "[Modelo de um bot do Discord pelo TypeScript]({{REF:links.template}})",
             "map": "https://kevinnovak.github.io/Time-Zone-Picker/",
-            "mapEmbed": "[Mapa de fuso horário]({{REF:links.map}})"
+            "mapEmbed": "[Mapa de Fusos Horários]({{REF:links.map}})"
         },
         "partials": {
             "congratulations": "🎉 Parabéns! 🎉",
             "yesOrNo": "`{{REF:yesNo.yes}}` ou `{{REF:yesNo.no}}`",
-            "italicizedDefaults": "Ajustes em *itálico* são padrões.",
+            "italicizedDefaults": "Configurações em *itálico* são as padrões.",
             "timeInTimeZone": "Em **{{TIME_ZONE}}** são **{{TIME_12_HOUR}}** / **{{TIME_24_HOUR}}**.",
             "findTimeZoneHelp": [
-                "Você pode validar o fuso horário em __{{REF:links.mapEmbed}}__.",
+                "Você pode achar fusos horários no __{{REF:links.mapEmbed}}__.",
                 "",
-                "Escolha o **select a location on the map** para validar o fuso horário."
+                "Basta **selecionar um local no mapa** para ver o nome de um fuso horário."
             ],
-            "translateNote": "Digite `{{REF:bot.prefix}} {{REF:commands.translate}}` para saber sobre as traduções.",
-            "dmWarning": "Alguns comandos precisam de permissão especial {{REF:bot.name}}."
+            "translateNote": "Digite `{{REF:bot.prefix}} {{REF:commands.translate}}` para saber mais sobre as traduções.",
+            "dmWarning": "Alguns comandos exigem mensagens diretas permitidas em pelo menos um servidor que você compartilha com o {{REF: bot.name}}."
```

Comment:
Remove the space in `{{REF: bot.name}}` --> `{{REF:bot.name}}`

Reply:
Done.